lib/src/showcase_widget.dart

```diff
@@ -152,6 +152,7 @@ class ShowCaseWidgetState extends State<ShowCaseWidget> {
   @override
   void initState() {
     super.initState();
+    anchoredOverlayKey = UniqueKey();
     initRootWidget();
```

Comment:
Just a small suggestion: you could directly initialize the variable in the field declaration instead of setting it in initState. This makes the code cleaner and easier to follow.

Reply:
@Sahil-Simform That makes sense, I have made the improvement.